src/main/scala/com/github/tkawachi/doctest/DoctestPlugin.scala

```diff
@@ -85,10 +69,10 @@ object DoctestPlugin extends AutoPlugin {
    */
   val doctestGenSettings = Seq(
     doctestTestFramework := (doctestTestFramework ?? ScalaCheck).value,
```

Comment:
The default value is a matter of preference and I've decided to leave as it was. I would say that ScalaTest would be a preferable choice just because ScalaTest is more popular (maybe? I don't really have any survey about that!). Again: in the end I've just left the way it was.